CHARTER.md

```diff
@@ -1,107 +1,121 @@
 
-- d. Participation in the Technical Initiative through becoming a Contributor, Collaborator, or Maintainer is open to anyone, whether a OpenSSF member or not, so long as they abide by the terms of this Technical Charter. 
+- f. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC. The TSC Chair will serve as the WG’s voting representative on the TAC. In the event that a proxy is requested by the TSC Chair, it may be any other TSC member so designated by a silent consensus of the TSC.
 
-- e. The TSC may create, change, modify, or remove roles or their definitions, so long as the definitions of roles for the Technical Initiative are publicly available in the Technical Initiative repository.
+- g. The current TSC Chair is:
 
-- f. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC.  **The TSC Chair, or any other TSC member so designated by the TSC, will serve as the Technical Initiative’s voting representative on the OpenSSF’s Technical Advisory Council (the "TAC").
+  - Eddie Knight, Sonatype (@eddie-knight)
 
-- g. Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Technical Initiative, which may include:
+## 3. Technical Initiatives
 
-   - i. coordinating the direction of the Technical Initiative;
+- a. A Technical Initiative is a project or special interest group (SIG) that falls under the oversight of this working group at creation or was subsequently added to this section of the charter.
+  
+  - No other work streams, activities, or bodies of any kind are considered Technical Initiatives, regardless of their affiliation with the WG.
 
-   - ii. approving, organizing or removing activities and projects;
+- b. Technical Initiatives must have a clearly identified lead who, for voting purposes, is not a lead on another ORBIT WG Technical Initiative.
 
-   - iii. establish community norms, workflows, processes, release requirements, and templates for the operation of the Technical Initiative;
+  - i. Leads will report to and participate in the routine activities of the TSC.
+  - ii. Leads may nominate their own replacement at any time, subject to the charter of the Technical Initiative.
+  - iii. In the event that the TSC elects to remove an acting lead, it must appoint a replacement in the same election.
 
-   - iv. establish a fundraising model, and approve or modify a Technical Initiative budget, subject to OpenSSF Governing Board approval;
+- c. A new Technical Initiative should only be brought into the WG after it is confirmed to be topically relevant, and interoperable with no less than two other Technical Initiatives in the WG. Acceptance must be confirmed on the proposal with a 2/3 majority approval from the Technical Steering Committee. That proposal must include an update to this document.
```

Comment:
Is "interoperable" defined? It's fairly generic. Perhaps an example of how the 3 current TIs meet this requirement could be definition enough.

Reply:
```suggestion
- c. A new Technical Initiative should only be brought into the WG after it is confirmed to be topically relevant, and interoperable with no less than two other Technical Initiatives in the WG. Acceptance must be confirmed on the proposal with a 2/3 majority approval from the Technical Steering Committee. That proposal must include an update to this document.

    - i. Interoperability is determined by the acceptance of outputs from one project as inputs to the core feature set of another.
    - ii. A project is considered interoperable if it accepts input from another project or if it provides output that is accepted as input to another project.

```